pkg/manager/member/pd_member_manager.go

```diff
@@ -522,7 +522,7 @@ func (pmm *pdMemberManager) pdStatefulSetIsUpgrading(set *apps.StatefulSet, tc *
 func getFailureReplicas(tc *v1alpha1.TidbCluster) int {
 	failureReplicas := 0
 	for _, failureMember := range tc.Status.PD.FailureMembers {
-		if failureMember.MemberDeleted {
+		if !failureMember.MemberDeleted {
```

Comment:
can you update this to reuse this function?
https://github.com/pingcap/tidb-operator/blob/master/pkg/manager/member/pd_failover.go#L101-L106